index.rb

```diff
@@ -0,0 +1,64 @@
+MORSE_CODE = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z',
+  '.----' => '1',
+  '..---' => '2',
+  '...--' => '3',
+  '....-' => '4',
+  '.....' => '5',
+  '-....' => '6',
+  '--...' => '7',
+  '---..' => '8',
+  '----.' => '9',
+  '-----' => '0',
+  '|' => ' '
+}.freeze
+
+def decode_char(letter)
+  MORSE_CODE[letter]
+end
+
+def decode_word(word)
+  letter = word.split
+  decoded_word = ''
+  letter.each do |l|
+    decoded_word += decode_char(l)
+  end
+  decoded_word += "\s"
+end
+
+def decode(code)
+  words = code.split('   ')
+  decoded_code = ''
+  words.each do |w|
+    decoded_code += decode_word(w)
+  end
+  puts decoded_code
```

Comment:
- [ ] [OPTIONAL] Kindly consider returning the decoded_code instead of using puts, in this case, because the exercise asks for returning the value not printing it

Reply:
Thanks :handshake: